nearswap/src/lib.rs

```diff
@@ -156,59 +157,50 @@ impl NearSwap {
     }
 
     /// Redeems `shares` for liquidity stored in this pool with condition of getting at least
-    /// `min_ynear` of Near and `min_tokens` of tokens. Shares are note
-    /// exchengable between different pools.
+    /// `min_ynear` of Near and `min_tokens` of tokens. Shares are not
+    /// exchagable between different pools.
     pub fn withdraw_liquidity(
         &mut self,
         token: AccountId,
-        shares: U128,
         min_ynear: U128,
         min_tokens: U128,
+        shares: U128,
```

Comment:
why reordering?

Reply:
was trying to keep order same as add_liquidity. will keep same it not required